plugins/modules/ssh_config.py

```diff
@@ -162,22 +162,264 @@
 import os
 import traceback
 
-from copy import deepcopy
-
-STORM_IMP_ERR = None
 try:
-    from storm.parsers.ssh_config_parser import ConfigParser
-    HAS_STORM = True
+    import paramiko
 except ImportError:
-    HAS_STORM = False
-    STORM_IMP_ERR = traceback.format_exc()
+    HAS_PARAMIKO = False
+    PARAMIKO_IMPORT_ERROR = traceback.format_exc()
+else:
+    HAS_PARAMIKO = True
+    PARAMIKO_IMPORT_ERROR = None
+
+from copy import deepcopy
+from operator import itemgetter
+import re
+
+from paramiko.config import SSHConfig
+import ansible.module_utils.six
+
+
+# Storm config parser from here: https://github.com/emre/storm/blob/master/storm/parsers/ssh_config_parser.py
+class StormConfig(SSHConfig):
```

Comment:
Could you please move all vendored code to `plugins/module_utils/_stormssh.py`? That way it is easier to declare code ownership and licensing correctly.